scripts/py_matter_yamltests/matter_yamltests/runner.py

```diff
@@ -162,7 +162,8 @@ async def _run_with_timeout(self, parser: TestParser, config: TestRunnerConfig):
         try:
             if config.auto_start_stop:
                 await self.start()
-            status = await asyncio.wait_for(self._run(parser, config), parser.timeout)
+            task = self._run(parser, config)
```

Comment:
Does a similar change need to happen in the python matter_testing_support file?

Reply:
Yes, the files should be 1:1 ... will update (was not aware they are there already)

Reply:
actually re-reading I see that is in wait_for but not about the copy of the file that @vatsalghelani-csa is working on (which should contain this change).

For matter_testing_support we may need this however we need to see where. I don't think we should just add shield everywhere without knowing what the sideffects are. Here this seems specific to a websocket runner.